examples/examples-log4j2/README.md

```diff
@@ -0,0 +1,62 @@
+grpc Log4j2 example
+==============================================
+
+The examples require grpc-java to already be built. You are strongly encouraged
+to check out a git release tag, since there will already be a build of grpc
+available. Otherwise you must follow COMPILING.md.
+
+You may want to read through the
+[Quick Start Guide](https://grpc.io/docs/quickstart/java.html)
+before trying out the examples.
+
+To build the examples,
+
+1. **[Install gRPC Java library SNAPSHOT locally, including code generation plugin](../../COMPILING.md) (Only need this step for non-released versions, e.g. master HEAD).**
+
+2. Run in this directory:
+```
+$ ../gradlew installDist
+```
+
+This creates the scripts `hello-world-server`, `hello-world-client`,
+`route-guide-server`, and `route-guide-client` in the
+`build/install/examples/bin/` directory that run the examples. Each
+example requires the server to be running before starting the client.
```

Comment:
This seems wrong. This only creates hello-world-logcontext-server and hello-world-logcontext-client and in build/install/examples-log4j2/bin directory?